harness/config/commands.yml

```diff
@@ -162,23 +162,33 @@ command('port <service>'):
     #!bash(workspace:/)|=
     passthru docker port "$($COMPOSE_BIN ps -q ={input.argument('service')})"
 
-command('set <attribute> <value>'):
+command('unset <attribute>'):
   env:
     ATTR_KEY: = input.argument('attribute')
-    ATTR_VAL: = input.argument('value')
   exec: |
     #!bash(workspace:/)|=
     if [ ! -f workspace.override.yml ]; then
       touch workspace.override.yml
     fi
-    if grep -q "attribute('${ATTR_KEY}'):" workspace.override.yml; then
+    if grep -q "^attribute('${ATTR_KEY}'):" workspace.override.yml; then
```

Comment:
Do you need `grep -E` for this to work across platforms?

Reply:
TBH I don't know, I was just looking at `sed` which is anchored but not used with `-E` either.

This says it's `^` is not an extended RE https://pubs.opengroup.org/onlinepubs/009696899/basedefs/xbd_chap09.html#tag_09_03_08 - only the use for inverting a character group is extended it seems.